Client/Frontend/Sync/BraveCore/History/Historyv2.swift

```diff
@@ -50,6 +50,8 @@ class Historyv2: WebsitePresentable {
     
     // MARK: Internal
     
+    public let historyNode: HistoryNode
```

Comment:
Same as above, why not move to use `HistoryNode` directly, add an extension on it for helpers

Reply:
Historyv2 is removed and HistoryNode extension is created with helper methods and usage integration